.env.test

```diff
@@ -2,3 +2,8 @@ GIT_API_TOKEN=test
 GIT_API_ENDPOINT=https://host.api/endpoint
 TTA_SERVICE_URL=http://tta-service.test
 PREVIEW_PAGES=1
+
+PUBLISHER_USERNAME=publisher
```

Comment:
We should put any non-local development secrets into Azure, even for the test site